### ISSUE TITLE
FieldSettings not being passed to field collection region.

### ISSUE THREAD

**Issue by sit-harry** [PiranhaCMS/piranha.core#2053]
Hi there!

It looks like field setting attributes aren't being passed to the field whenever it is a single field collection. 

If we have the below example...

```cs
[PageType(Title = "Standard page")]
public class StandardPage : Page<StandardPage>
{
    [Region]
    [StringFieldSettings(DefaultValue = "Hello World!")]
    public List<StringField> Fields { get; set; }

    [Region]
    public TestRegion TestRegion { get; set; }
}

public class TestRegion
{
    [Field]
    public StringField RegularStringField { get; set; }

    [Field]
    [StringFieldSettings(DefaultValue = "Hello World!")]
    public StringField FancyStringField { get; set; }
}
```

The `FancyStringField` works as expected, with the settings being pushed through to the field. 

![FieldWithinRegion](https://github.com/PiranhaCMS/piranha.core/assets/139478417/81246936-5120-46f1-9f4f-194ce63a1976)

However, with the `Fields` region, the settings aren't being pushed through. 

![FieldCollection](https://github.com/PiranhaCMS/piranha.core/assets/139478417/33770ccd-a6eb-4c85-9b27-467940f6aca7)

It looks like the settings needs to be added to the default `ContentTypeField` [here](https://github.com/PiranhaCMS/piranha.core/blob/be0c6da3d99e21078aeef43e4f4005e7a661cc50/core/Piranha.AttributeBuilder/ContentTypeBuilder.cs#L619C53-L619C54) and then pulled through [here](https://github.com/PiranhaCMS/piranha.core/blob/be0c6da3d99e21078aeef43e4f4005e7a661cc50/core/Piranha.Manager/Services/ContentTypeService.cs#L269).

Will try to get a PR together soon, but let me know if I'm missing anything here, thanks :)

**Comment by tidyui** [comment]
I simply just don’t think this case is covered by the settings. I’ll tag this as a feature request!

**Comment by tidyui** [comment]
Fixed as of #2054
